src/main/java/bio/terra/cloudfunctions/streaming/GcsBQ.java

```diff
@@ -20,6 +37,87 @@
 
 public class GcsBQ implements BackgroundFunction<GCSEvent> {
   private static final Logger logger = Logger.getLogger(GcsBQ.class.getName());
+  private static final String DATASET = "simple_stream_dataset";
+  private static final String TABLE = "SUMMARY_testRun";
+  private static final Schema SCHEMA =
+      Schema.of(
```

Comment:
Is there possibly a way to make this schema builder more readable?

Reply:
The schema json file is now in the `resources` directory. The GHA has been refactored to create the schema using `bq` cli. The `GcsBQ` code has been refactored so that it is not responsible for creating the `BQ` table.